luxtronik/__init__.py

```diff
@@ -159,10 +197,8 @@ def _write(self, parameters):
         parameters.queue = {}
         # Give the heatpump a short time to handle the value changes/calculations:
         time.sleep(WAIT_TIME_AFTER_PARAMETER_WRITE)
```

Comment:
I think this line should be moved to the `_write_and_read` function. We do not need to wait if the user only wants to write.

Reply:
I would leave it there in case the user write:
```python
lux.write()
lux.read_parameters()
```

Reply:
But if the user wants to write and read, he can call `write_and_read()`.

Reply:
Done.

Alternatively, we could implement something like:
```python
def write():
...
    timeout = datetime.now() + timedelta(0,10)

def _lock_and_connect():
    sleep = (timeout - datetime.now()).total_seconds()
    if sleep > 0:
        time.sleep(sleep)
    ...
```